index.html

```diff
@@ -37,34 +36,206 @@ <h2 class="logo">My Logo</h2>
 
     <section class="headline">
       <div class="intro">
-        <h2 style="font-size: clamp(30px, 4vw, 45px); font-weight: 700;">
+        <h2 id="h-tag">
           I’m Fatihah <br />
           Glad to see you!
         </h2>
       </div>
       <div class="bio">
-        <p style="font-size: clamp(16px, 4vw, 20px);">
-          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
+        <p id="p-tag1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
       </div>
       <div class="socials">
-        <h4 class="h6">LET'S CONNECT</h4>
-        <ul>
-          <ol>
+        <h4 class="letsconnect">LET'S CONNECT</h4>
+        <ul class="socials-mainlist">
+          <ol class="socials-list">
             <a href="#"><img src="./icons/github.png" alt="github icon" /></a>
           </ol>
-          <ol>
+          <ol class="socials-list">
             <a href="#"><img src="./icons/linkedin.png" alt="linkedin icon" /></a>
           </ol>
-          <ol>
+          <ol class="socials-list">
             <a href="#"><img src="./icons/twitter.png" alt="twitter icon" /></a>
           </ol>
-          <ol>
+          <ol class="socials-list">
             <a href="#"><img src="./icons/medium.png" alt="medium icon" /></a>
           </ol>
         </ul>
       </div>
     </section>
+    <br />
+    <section id="tonic1" class="work-cards">
+      <div class="work-img"><img class="work-img" src="icons/tonic1.png" alt="" /></div>
+      <h2 id="h-tag" class="tonic">Tonic</h2>
+      <div>
+        <ul class="canopy-list">
+          <ol class="canopy">
+            CANOPY
+          </ol>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <div class="aboutme-intro">
+        <p id="p-tag">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <ul class="small-langul">
+        <li class="small-langli">html</li>
+        <li class="small-langli">css</li>
+        <li class="small-langli">javascript</li>
+      </ul>
+      <div>
+        <button class="seeproject active">See Project</button>
+      </div>
+    </section>
```

Comment:
- Please use only one `<section>` tag to wrap your works section🙏. This is for semantic reasons as section tags are used to mark different parts of a webpage. You can use a `<div>` tag instead to wrap each individual card.
- After adding the section tag please make sure to use `display: grid` to style the works sections as it is one of the project requirements 
<img width="774" alt="image" src="https://user-images.githubusercontent.com/37116322/219852567-108f0e52-461e-4a93-81cd-894cb47b025c.png">

Currently, you use `display: grid` in your code, but it is not being applied anywhere